proto/cacheclient.proto

```diff
@@ -175,3 +185,105 @@ message _SetDifferenceResponse {
     _Missing missing = 2;
   }
 }
+
+// stored = request + stored
+message _ListPushFrontRequest {
+  bytes list_name = 1;
+  repeated bytes values = 2;
+  uint64 ttl_milliseconds = 3;
+  bool refresh_ttl = 4;
+}
+
+message _ListPushFrontResponse {}
+
+// stored = stored + request
+message _ListPushBackRequest {
+  bytes list_name = 1;
+  repeated bytes values = 2;
+  uint64 ttl_milliseconds = 3;
+  bool refresh_ttl = 4;
+}
+
+message _ListPushBackResponse {}
+
+message _ListPopFrontRequest {
+  bytes list_name = 1;
+}
+
+message _ListPopFrontResponse {
+  message _Found {
+    bytes front = 1;
+  }
```

Comment:
Can we easily change this to support a request to pop `count` items from a list `list_name`?

Reply:
Good callout. @kvcache we should make the change on the blobdb server as well.

Reply:
@malandis going to leave this as is for now, should be easy to add this functionality later.